primitives/Cargo.toml

```diff
@@ -5,10 +5,10 @@ authors = ["Centrality Developers <support@centrality.ai>"]
 edition = "2018"
 
 [dependencies]
-codec = { package = "parity-scale-codec", version = "1.0.0", default-features = false, features = ["derive"] }
-primitives = { package = "sp-core", git = "https://github.com/plugblockchain/plug-blockchain", default-features = false, branch = "1.0.0-rc2" }
+parity-scale-codec = { version = "1.0.0", default-features = false, features = ["derive"] }
```

Comment:
My bad for not clarifying this but could we keep the old `codec = { package = "parity-scale-codec", version = "1.0.0", default-features = false, features = ["derive"] }` <- this is used throughout plug